doc/tutorials/12-constant_pH/12-constant_pH.ipynb

```diff
@@ -166,95 +314,254 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "num_samples = 100\n",
+    "# the reference data from Henderson-Hasselbalch equation\n",
+    "def ideal_alpha(pH, pK):\n",
+    "    return 1. / (1 + 10**(pK - pH))\n",
     "\n",
-    "pHs = np.linspace(1, 8, num=15)\n",
-    "degrees_of_dissociation = []\n",
-    "std_dev_degree_of_dissociation = []\n",
-    "histograms = []\n",
-    "histogram_edges = range(0,N0+1)\n",
+    "# empty lists as placeholders for collecting data\n",
+    "numAs = [] # number of A- species observed at each sample\n",
```

Comment:
please come up with a more expressive variable name which maybe makes it possible to get rid of the comment:
*e.g.* ```num_of_A```?

Reply:
I made the variable name more explicit but still prefer to keep the comment.